From 56cfd8d36bdf98af82a302d811e9b01caaff2679 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 21 Mar 2017 22:01:43 +0100
Subject: [PATCH] calendar-server: Remove unused variables and function

The code that used them was removed in commit 7e0e224e0eb8.
---
 src/calendar-server/gnome-shell-calendar-server.c | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/src/calendar-server/gnome-shell-calendar-server.c b/src/calendar-server/gnome-shell-calendar-server.c
index 57bc71bb8..b607b7331 100644
--- a/src/calendar-server/gnome-shell-calendar-server.c
+++ b/src/calendar-server/gnome-shell-calendar-server.c
@@ -387,21 +387,6 @@ calendar_appointment_init (CalendarAppointment  *appointment,
                                                    default_zone);
 }
 
-static icaltimezone *
-resolve_timezone_id (const char *tzid,
-                     ECalClient *source)
-{
-  icaltimezone *retval;
-
-  retval = icaltimezone_get_builtin_timezone_from_tzid (tzid);
-  if (!retval)
-    {
-      e_cal_client_get_timezone_sync (source, tzid, &retval, NULL, NULL);
-    }
-
-  return retval;
-}
-
 static CalendarAppointment *
 calendar_appointment_new (icalcomponent        *ical,
                           ECalClient           *cal)
@@ -624,7 +609,6 @@ app_load_events (App *app)
     {
       ECalClient *cal = E_CAL_CLIENT (l->data);
       GError *error;
-      GSList *objects, *j;
       ECalClientView *view;
       CollectAppointmentsData data;
 
-- 
2.12.0

